vncviewer/touch.cxx

```diff
@@ -314,44 +314,15 @@ void FLTKGestureHandler::fakeButtonEvent(bool press, int button,
                                          const GHEvent origEvent)
 {
   XEvent fakeEvent;
-  int mask;
 
   memset(&fakeEvent, 0, sizeof(XEvent));
 
   fakeEvent.type = press ? ButtonPress : ButtonRelease;
   fakeEvent.xbutton.button = button;
   preparePointerEvent(&fakeEvent, origEvent);
 
-  switch (button) {
-    case Button1:
-      mask = Button1Mask;
-      break;
-    case Button2:
-      mask = Button2Mask;
-      break;
-    case Button3:
-      mask = Button3Mask;
-      break;
-    case Button4:
-      mask = Button4Mask;
-      break;
-    case Button5:
-      mask = Button5Mask;
-      break;
-    case Button6:
-      mask = Button6Mask;
-      break;
-    case Button7:
-      mask = Button7Mask;
-      break;
-    default:
-      mask = 0;
-  }
-
-  if (press)
-    fakeEvent.xbutton.state |= mask;
-  else
-    fakeEvent.xbutton.state &= ~mask;
+  if (!press)
```

Comment:
Did you test pressing multiple buttons at the same time? ;)

Reply:
How?